src/components/explore/AggregationsOverTimeGraph.svelte

```diff
@@ -178,20 +186,32 @@
 
     // get percentile data of linear and log graphs
     if (
-      data[0].metric_type === 'histogram-linear' ||
-      data[0].metric_type === 'quantity' ||
+      linearMetrics.includes(data[0].metric_type) ||
       yScaleType === 'scalePoint'
     ) {
+      // do not change yDomain when all percentiles are selected
+      // (we want the graph to match the violin plot initially)
+      if ($store.visiblePercentiles.length === 5) return yDomain;
```

Comment:
We were not able to rescale the y axis because we used to have the same set of data for yDomain regardless of toggled percentiles. In my last change, I calculated the yDomain range to only include the highest and smallest values available of the shown percentile lines, so that this array will dynamically change if a percentile is toggled. However this means that the range is more tightly scoped than before, and you can see the difference in initial behavior here: [dev](https://dev.glam.nonprod.dataops.mozgcp.net/firefox/probe/time_to_non_blank_paint_ms/explore?) vs [stage](https://stage.glam.nonprod.dataops.mozgcp.net/firefox/probe/time_to_non_blank_paint_ms/explore?).

@rafrombrc wanted the initial range of the graph to match the violin plot graph, so I added this condition here to preserve the yDomain range if all percentile lines are visible (i.e no percentiles toggled yet)

Below you can see that the yDomain range matches the violin plot initially.

![match](https://user-images.githubusercontent.com/28797553/122987601-a2168d80-d355-11eb-97eb-bac1a43cdd2b.gif)